examples/asr/speech_to_text.py

```diff
@@ -35,12 +35,21 @@
 
 Hydra logs will be found in "$(./outputs/$(date +"%y-%m-%d")/$(date +"%H-%M-%S")/.hydra)"
 PTL logs will be found in "$(./outputs/$(date +"%y-%m-%d")/$(date +"%H-%M-%S")/lightning_logs)"
+
+ python speech_to_text.py AudioToTextDataLayer.manifest_filepath="./an4/train_manifest.json" AudioToTextDataLayer_eval.manifest_filepath="./an4/test_manifest.json" hydra.run.dir="." pl.trainer.gpus=2 pl.trainer.max_epochs=100
+
+Override some args of optimizer
+ python speech_to_text.py AudioToTextDataLayer.manifest_filepath="./an4/train_manifest.json" AudioToTextDataLayer_eval.manifest_filepath="./an4/test_manifest.json" hydra.run.dir="." pl.trainer.gpus=2 pl.trainer.max_epochs=2 optim.args.params.betas=[0.9,0.22] optim.args.params.weight_decay=0.0000005
+
+Overide optimizer entirely
+ python speech_to_text.py AudioToTextDataLayer.manifest_filepath="./an4/train_manifest.json" AudioToTextDataLayer_eval.manifest_filepath="./an4/test_manifest.json" hydra.run.dir="." pl.trainer.gpus=2 pl.trainer.max_epochs=2 optim.name=adamw optim.lr=0.001 ~optim.args +optim.args.betas=[0.9,0.22] +optim.args.weight_decay=0.0000005
+
```

Comment:
Can we format this similar to what is already there?

Reply:
Ah my bad this got merged in. This was just convenience for me to test out the scripts, they will be removed.